R/chol_solve_linear_system.R

```diff
@@ -0,0 +1,7 @@
+# solves Ax = b for x
```

Comment:
For consistency, I'd use roxygen style function comments everywhere, even if you're not exporting them.